scripting/get5.sp

```diff
@@ -1504,18 +1482,7 @@ public void WriteBackup() {
   g_RoundBackupPathCvar.GetString(folder, sizeof(folder));
   ReplaceString(folder, sizeof(folder), "{MATCHID}", g_MatchID);
 
-  int backupFolderLength = strlen(folder);
-  if (backupFolderLength > 0 &&
-      (folder[0] == '/' || folder[0] == '.' || folder[backupFolderLength - 1] != '/' ||
-       StrContains(folder, "//") != -1)) {
-    LogError(
-        "get5_backup_path must end with a slash and must not start with a slash or dot. It will be reset to an empty string! Current value: %s",
-        folder);
-    folder = "";
-    g_RoundBackupPathCvar.SetString(folder, false, false);
-  } else {
-    CreateBackupFolderStructure(folder);
-  }
+  CheckAndCreateFolderPath(g_RoundBackupPathCvar);
```

Comment:
You're replacing `{MATCHID}` in `folder` but then you pass the Cvar. You won't have replacement then.